src/storybook/components/related-components/descriptions/chips-description.jsx

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from "react";
+import { RelatedComponent } from "../../related-component/related-component";
+import Chips from "../../../../components/Chips/Chips";
+
+export const ChipsDescription = () => {
+  const component = useMemo(() => <Chips label="This is a chip" readOnly={true} />, []);
```

Comment:
please use shorthand syntax for set a true (boolean) value to a prop, such in the following example: `<Checkbox label="Selected" checked />,`